keys/keys.go

```diff
@@ -246,9 +246,9 @@ func RangeMetaKey(key proto.Key) proto.Key {
 	return proto.KeyMin
 }
 
-// ValidateRangeMetaKey validates that the given key is a valid Range Metadata
+// validateRangeMetaKey validates that the given key is a valid Range Metadata
 // key.
```

Comment:
add to the comment: This checks only the constraints common to forward and backwards scans: correct prefix and not exceeding `KeyMax`.